mrp_production_category_notes/README.rst

```diff
@@ -0,0 +1,77 @@
+=============================
+MRP Production Category Notes
+=============================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:028e533d1fd512d6f7003577017d7eecb8b9056696f7e01324dd6f4b45d5463f
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-sygel--technology%2Fsy--manufacture-lightgray.png?logo=github
+    :target: https://github.com/sygel-technology/sy-manufacture/tree/15.0/mrp_production_category_notes
+    :alt: sygel-technology/sy-manufacture
+
+|badge1| |badge2| |badge3|
+
+This module adds a new field to product categories that allows
+adding notes to manufacturing orders.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Go to apps
+#. Look for mrp_production_category_notes module
+#. Click install
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Inventory / Configuration / Products / Products categories / Category
```

Comment:
You need to give a more detailed explanation. Explain the name of the field that needs to be completed and how the value of this field is transfered to manufacturing orders.